src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6859,6 +6859,12 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_CollectionExpressionTargetNoElementType" xml:space="preserve">
     <value>Collection expression target '{0}' has no element type.</value>
   </data>
+  <data name="ERR_CollectionExpressionMissingConstructor" xml:space="preserve">
```

Comment:
I found tests for missing ctor, but couldnt' find tests for inaccessible constructors. Do we have some? I wonder if this error message should mention the accessibility requirement.
The same question may apply for the error message below (for `Add` method)

Reply:
Added some specific tests for inaccessible constructor and `Add`. The existing errors reported for those cases seem reasonable to me.